build.gradle.kts

```diff
@@ -14,6 +14,8 @@ repositories {
 
 dependencies {
 	implementation("org.springframework.boot:spring-boot-starter")
+	implementation("org.springframework.boot:spring-boot-starter-web")
+	implementation("org.springframework.boot:spring-boot-starter-actuator")
```

Comment:
Ah, this provides /health and /info endpoints, for the k8s liveness & readiness checks

Reply:
Yes, currently we are using /health endpoint for both checks
<img width="1090" alt="Screenshot 2023-06-07 at 15 27 34" src="https://github.com/ministryofjustice/cfe-crime/assets/195928/95bc4c84-d9ec-4d67-8ece-6b8e2848ec3d">